mandatory/1-fix-functions.js

```diff
@@ -21,18 +20,18 @@ function mood() {
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
+  let isBigEnough=10;
 
-  if (isBigEnough) {
+  if (isBigEnough<num) {
     return "num is greater than 10";
   } else {
     return "num is not big enough";
   }
 }
 
 function get3rdIndex(arr) {
-  let index = 3;
-  let element;
+  let index = 2;
```

Comment:
From looking at the test cases at the bottom of the file, this is expecting `arr[3]` rather than `arr[2]` - you have changed the index to 2, which will give you the third element (I can see your logic!) but that isn't what the test is expecting to see.

Reply:
(you can also one-line solve this if you don't set up named variables for things like `index` and `element`; in this particular case, since the method name is clearly specifically going to target the 3rd index, and it's not a value that will ever vary, I'd consider something like:

```
function get3rdIndex(arr) { 
    return arr[3]
}
```
)